lib/html_types.mli

```diff
@@ -2002,6 +2002,7 @@ type form_attrib =
     | `Method
     | `Name
     | `Target
+    (* TODO [aerben] forms only allow `On and `Off, how to deal with that? *)
```

Comment:
To answer your question: What's the behavior of web browser if you use the token list in a form ? Given what I know of html rendering, it's probably accepted silently, so we might as well allow it.

Reply:
I check in one of my own forms in Chromium (Version 102.0.5005.115) and if it's a token list it seems to default to `"on"`. Omitting the attribute also defaults to `"on"` in my browser (so I'm assuming it is, as you said, accepted silently).